test/algo/compression/v2_5/utils/bert_ds_config.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
do we still have folder v2_5?   @J-shang

Reply:
no, this folder will be replaced to `test/algo/compression`

Reply:
great! will be replaced in the following pr, right?

Reply:
yes, in pr #5573